src/styles/product.scss

```diff
@@ -0,0 +1,125 @@
+.products {
+  position: relative;
+  display: grid;
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @include on-md-mobile {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-desktop {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product {
+  // width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
Not resolved